### ISSUE TITLE
Implement a message handler between UI and main.js

### ISSUE THREAD

**Issue by LukeFinch** [gavinmcfarland/plugma#9]
I feel like this would be a good option to have in the templates, for a quick starter.

[Figma's AI template](https://github.com/figma/ai-plugin-template/blob/main/lib/figmaAPI.ts) has an interesting implementation.

I know we've tried other implementations in the past, such as the ones in my Vue3 template, and create-figma-plugin

**Comment by gavinmcfarland** [comment]
I agree. I've been thinking about this and I have something planned. Stay tuned.

**Comment by svallory** [comment]
Hey, take a look at the [message bus](https://github.com/cva-design/figma-sdk/tree/feature/message-bus/packages/message-bus) (almost done) and the [RPC](https://github.com/cva-design/figma-sdk/tree/main/packages/rpc) packages I created for the figma-sdk
